src/main/java/seedu/address/logic/commands/AttendanceCommand.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.logic.commands;
```

Comment:
Good job Jeun Hoe! Just to check with you, will you be implementing the saving of data into json after v1.2?